src/components/LandingPage/landingPage.test.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import LandingPage from './LandingPage';
+
+describe('<landingPage />', () => {
+  test('it should mount', () => {
+    render(<LandingPage />);
+    
+    const landingPage = screen.getByTestId('LandingPage');
+
+    expect(landingPage).toBeInTheDocument();
```

Comment:
Thanks for taking care of this test! Should we test that the button causes the main page to render?

Reply:
Or is this something that is good enough to test manually? Now that I think about it I'm sure that's the case, but want to double check in case.

Reply:
let me see whether there's an easy why. If not, I don't think we should spend extra time on it

Reply:
I agree. After I looked at this one I wondered the same thing about the help button and googled testing hover buttons and the response was basically 'why bother when you can test it manually easily' and I think that applies to this situation too

Reply:
So what's our plan, should we remove this test function?

Reply:
I just reviewed the code, there's no easy way we can test it since the onclick is from the parent actually and everything in this component is in render(). So I think we can merge this. Answer @Jaega 's question. I think we are trying to aim for a minimum test (whether it can render) for every component, so here maybe we just test whether it can render and that's all. I believe since everything's in render(), it might show a 100% coverage for this file actually.

Reply:
I think there's a way to do it with mocks so I can give it a try later on, but for now we can go ahead and run with this test.

Reply:
feel free to make any changes if you think you can find a way to make that work. Thanks!